libs/community/langchain_community/embeddings/ovhcloud.py

```diff
@@ -51,9 +51,11 @@ def _generate_embedding(self, text: str) -> List[float]:
         """
         headers = {
             "content-type": "text/plain",
-            "Authorization": f"Bearer {self.access_token}",
         }
 
+        if self.access_token is not None:
+            headers["Authorization"] = f"Bearer {self.access_token}"
```

Comment:
what happens to requests if they are missing this header?

Reply:
They are just rate-limited to 12 calls/min.

Reply:
As it is enforced now, I resolve the conversation.